File: AniDAAPI/AniDAAPI/Models/Customer.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class Customer
    {
        public int CustomerId { get; set; }
        public String CustomerName { get; set; }
        public String CustomerRef { get; set; }
        public bool isActive { get; set; }
        public int LastUpdatedBy { get; set; }
        public DateTime LastUpdatedDt { get; set; }
    }
}

File: DA_CustomerWebPortal/lib/Model/Customer.dart
class Customer {
  int customerID;
  String customerName;
  String customerRef;
  bool isActive;
  int lastUpdatedBy;
  String lastUpdatedDT;

  Customer({
    this.customerID,
    this.customerName,
    this.customerRef,
    this.isActive,
    this.lastUpdatedBy,
    this.lastUpdatedDT,
  });

  factory Customer.fromJson(Map<String, dynamic> json) {
    return Customer(
      customerID: json['customerId'],
      customerName: json['customerName'],
      customerRef: json['customerRef'],
      isActive: json['isActive'],
      lastUpdatedBy: json['lastUpdatedBy'],
      lastUpdatedDT: json['lastUpdatedDt'],
    );
  }
}
